openml/runs/run.py

```diff
@@ -139,8 +139,37 @@ def predictions(self) -> pd.DataFrame:
     def id(self) -> Optional[int]:
         return self.run_id
 
+    def evaluation_summary(self, metric: str):
+        """Summarizes the evaluation of a metric over all folds.
+
+        The fold scores for the metric must exist already. During run creation,
+        by default, the MAE for OpenMLRegressionTask and the accuracy for
+        OpenMLClassificationTask/OpenMLLearningCurveTasktasks are computed.
+
+        If repetition exist, we take the mean over all repetitions.
+
+        Parameters
+        ----------
+        metric: str
+            Name of an evaluation metric that was used to compute fold scores.
+
+        Returns
+        -------
+        metric_summary: str
+            A formatted string that displays the metric's evaluation summary.
+            The summary consists of the mean and std.
+        """
+        fold_score_lists = self.fold_evaluations[metric].values()
+
+        # Get the mean and std over all repetitions
+        rep_means = [np.mean(list(x.values())) for x in fold_score_lists]
+        rep_stds = [np.std(list(x.values())) for x in fold_score_lists]
+
+        return "{:.4f} +- {:.4f}".format(np.mean(rep_means), np.mean(rep_stds))
```

Comment:
Does this work for all kinds of tasks?

Reply:
Good question, as far as I can tell, the fold evaluations should be stored in such a "list of lists" container for all tasks. 

But I have not found an overview for all tasks and split settings to confirm this. I could a test for that. 
Do you know the conventions for this, or is this specified somewhere?

Reply:
We store the task types `openml.tasks.task`, and we might store the evaluation protocols somewhere, too. Then you could query the server for runs that follow the cross-product of the two and check those.

Reply:
I will change the design such that the current PR only enables the feature for supported task types (SUPERVISED_CLASSIFICATION, LEARNING_CURVE, SUPERVISED_REGRESSION). 

Moreover, I will add a test for the print and string representation.

Reply:
This will now only be for local measures where we also have fold evaluations. Specifically, this now only aggregates the fold scores computed here: https://github.com/openml/openml-python/blob/develop/openml/runs/functions.py#L582

Reply:
Did you already do so?

Reply:
Yes, it did not change the code here, but where the function is called. We could resolve this conversation if you are okay with it.

Reply:
Yes, fine with me.